docs/blog/2025-q3-changelog.mdx

```diff
@@ -0,0 +1,139 @@
+---
+slug: 2025-q3-updates-and-changes
+title: 2025 Q3 Updates and Changes
```

Comment:
This is being pedantic, but this should be in the Q4 update. I'd be ok with adding both the Q3 and Q4 updates in this PR and then just updating the Q4 one at the end of the year. 

Alternatively, we could simply switch to an "H2" update? Any preferences?

Reply:
I like the idea of splitting it into Q3 and Q4

Reply:
7234794